products/models.py

```diff
@@ -0,0 +1,40 @@
+from django.db          import models
+
+from categories.models  import SecondCategory
+from core.models        import TimeStampModel
+
+class Type(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'types'
+
+class Product(TimeStampModel):
+    title           = models.CharField(max_length=45)
+    description     = models.TextField()
+    second_category = models.ForeignKey(SecondCategory, on_delete=models.CASCADE)
+    price           = models.DecimalField(decimal_places=3, max_digits=10)
+    stock           = models.IntegerField()
+    discount        = models.DecimalField(null=True, decimal_places=0, max_digits=3)
+    type            = models.ManyToManyField('Type')
```

Comment:
```suggestion
    types            = models.ManyToManyField('Type')
```
ManyToManyField를 사용하는 경우 변수명을 복수로 지어주세요!
```python
product = Product.objects.get(id=..)
product.types (o)
product.type (x)
```

Reply:
types로 수정했습니다.